index.bs

```diff
@@ -6022,12 +6022,15 @@ There MUST NOT be any values returned for ignored extensions.
 
 # Defined Extensions # {#sctn-defined-extensions}
 
-This section defines an additional set of extensions to be registered in the
+This section and it's subsections defines an additional set of extensions to be registered in the
 IANA "WebAuthn Extension Identifiers" registry [[!IANA-WebAuthn-Registries]] established by [[!RFC8809]].
 These MAY be implemented by user agents targeting broad interoperability.
 
+## Client Extensions
 
-## FIDO <dfn>AppID</dfn> Extension (appid) ## {#sctn-appid-extension}
+This section defines extensions that are only [=client extensions=].
+
+### FIDO <dfn>AppID</dfn> Extension (appid) ## {#sctn-appid-extension}
```

Comment:
Ah right, you need to pad the header mark on both sides...
(this becomes very apparent in the [preview](https://pr-preview.s3.amazonaws.com/Firstyear/webauthn/pull/1762.html), which gets auto-added to the OP.)

```suggestion
### FIDO <dfn>AppID</dfn> Extension (appid) ### {#sctn-appid-extension}
```